core/contracts/primitive_stream.kf

```diff
@@ -323,7 +323,14 @@ procedure get_index($date_from text, $date_to text, $frozen_at int, $base_date t
         value decimal(36,18)
         ) {
 
-    $baseValue decimal(36,18) := get_base_value($base_date, $frozen_at);
+    $effective_base_date text := $base_date;
+    if ($effective_base_date == '') {
+        for $v_row in SELECT * FROM get_metadata('default_base_date', true, null) ORDER BY created_at LIMIT 1 {
```

Comment:
```suggestion
        for $v_row in SELECT * FROM get_metadata('default_base_date', true, null) ORDER BY created_at DESC LIMIT 1 {
```

Reply:
oh i'm sorry, i did write it on the other contract from other repo, but i missed it here. thank you for reminding me